job-view.php

```diff
@@ -1,91 +1,122 @@
+                            <div class="job-meta">
+								<?php
+								if ( ! empty( $job['company_site'] ) ) { ?>
+                                    <a class="meta-company"
+                                       href="<?php echo 'http://' . $job['company_site'] ?>"><?php echo $job['company_name'] ?></a>
+								<?php } else {
+									echo $job['company_name'];
+								}
+								?>
+                                <span class="meta-date">
                                         <?php
-                                        if (date_difference($row['publication_date']) < 31) {
-                                            echo "Posted " . date_difference($row['publication_date']) . " days ago";
+                                        if ( date_difference( $job['publication_date'] ) < 31 ) {
+	                                        echo "Posted " . date_difference( $job['publication_date'] ) . " days ago";
                                         } else {
-                                            echo "Posted over 1 month ago";
+	                                        echo "Posted over 1 month ago";
                                         }
                                         ?>
                                     </span>
-                                </div>
-                                <div class="job-details">
+                            </div>
+                            <div class="job-details">
                                     <span class="job-location">
                                         <?php
-                                        if (!empty($row['jobs_location'])) {
-                                            echo "Job location: " . $row['jobs_location'];
+                                        if ( ! empty( $job['jobs_location'] ) ) {
+	                                        echo "Job location: " . $job['jobs_location'];
                                         } else {
-                                            echo "No location specified";
+	                                        echo "No location specified";
                                         }
                                         ?>
                                     </span>
-                                    <span class="job-type">Contract staff:</span>
-                                    <span class="job-price"><?php echo $row['salary']; ?> лв.</span>
-                                </div>
-                            </header>
-                            <div class="job-body">
-                                <?php
-                                echo $row['description'];
-                                ?>
+                                <span class="job-type">Contract staff:</span>
+                                <span class="job-price"><?php echo $row['salary']; ?> лв.</span>
                             </div>
+                        </header>
+                        <div class="job-body">
+							<?php
```

Comment:
formatting alert

Reply:
I reformatted it. Not sure if this should be the end result but i think it's better.